terraform/README.md

```diff
@@ -36,14 +40,14 @@ Create integrations, for instance:
 
 ```text
 resource "juju_integration" "amf-nrf" {
-  model = var.model_name
+  model = var.model
```

Comment:
Following the previous comment, why not use `juju_model.my_model.name`?

Reply:
Done, thanks!